api/v1_metrics_total_artists.go

```diff
@@ -0,0 +1,18 @@
+package api
+
+import (
+	"github.com/gofiber/fiber/v2"
+)
+
+func (app *ApiServer) v1MetricsTotalArtists(c *fiber.Ctx) error {
+	var total int64
+	if err := app.pool.QueryRow(c.Context(), `
+        SELECT COUNT(*)::bigint
+        FROM aggregate_user
+        WHERE total_track_count > 0
+    `).Scan(&total); err != nil {
+		return err
+	}
+
+	return c.JSON(fiber.Map{"data": map[string]int64{"total": total}})
```

Comment:
I guess I'm on a bigint kick lately.

This feels slightly off, as JSON can't represent an int64. I wonder if it just lets it truncate if you exceed the limits.
Supposedly, it should serialize as a string if you want to represent an int64.